source/MaterialXRuntime/RtApi.cpp

```diff
@@ -50,15 +51,33 @@ void RtApi::initialize()
     registerTypedSchema<RtNodeDef>();
     registerTypedSchema<RtNodeGraph>();
     registerTypedSchema<RtBackdrop>();
-    registerTypedSchema<RtLookGroup>();
-    registerTypedSchema<RtLook>();
-    registerTypedSchema<RtMaterialAssign>();
-    registerTypedSchema<RtCollection>();
+    registerTypedSchema<RtLookGroup, RtLookGroupConnectableApi>();
+    registerTypedSchema<RtLook, RtLookConnectableApi>();
+    registerTypedSchema<RtMaterialAssign, RtMaterialAssignConnectableApi>();
+    registerTypedSchema<RtCollection, RtCollectionConnectableApi>();
```

Comment:
Add overrides for the connectable API for the look management nodes, to handle validation of what is allowed to connect to them.